app/templates/ui/styles/main.less

```diff
@@ -188,12 +188,6 @@ footer {
     background: url(/images/MarkLogic-Powered-By.png) 10px 10px no-repeat transparent;
   }
 
-/*Content well*/
-.content {
-  background-color: #fff;
-  padding: 20px 0;
-}
-
```

Comment:
Something was removed from the general main.less file. Could that harm the other templates?